### PR TITLE
added Azure .NET SDK to README

### PR DESCRIPTION
**Pull request by crystaju**
provided the Azure .NET SDK in the tools section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ For a longer list, see:
 For a list of free services and tools for open source .NET projects, see:
 
 * [Free Services & Tools for Open Source .NET Projects](dotnet-free-oss-services.md)
+* [Download the latest Azure .NET SDK](https://www.visualstudio.com/features/azure-tools-vs)
```
Comment (svick) [inline README.md]: I don't think the link belongs here, since it's not a "list of free services and tools for open source .NET projects".

Comment (crystaju) [inline README.md]: I have a different thread with Lee Coward on this. It is not open source projects, but it is free tools and services to help on any .NET projects.

**Comment by richlander** [comment]
We could do two things:

- Add the Azure SDK repos to the open source library list.
- Add the specific free Azure services to the free services list.

That would seem to align with the intentions of both lists. Putting the Azure SDK link in the specific place you did doesn't quite make sense. We could create another category on this page that the Azure SDK would fit into but I'm not quite sure what it would be. Maybe "Popular .NET SDKs"?

**Comment by modulexcite** [comment]
How about a category for all things cloud?

**Comment by crystaju** [comment]
I would love to see a section for all the cloud related resources, or popular .NET SDKs! However, I am not familiar with other tools, because I am approaching the question from helping developers attaching to Azure in general. If any of you can provide that information, I am happy to help putting the details in. Also, I am working with Lee Coward <leecow@microsoft.com> on an internal thread.

**Pull request closed without merging**